docs/content/cli.md

```diff
@@ -196,6 +197,33 @@ Global Flags:
       --debug   Enable debug logging
 ```
 
+### Bundles List
+
+This command is available both as `porter bundle list` and `porter bundles list`.
+
+```console
+ $ porter bundle list --help
+List all bundles installed by Porter.
+
+A listing of bundles currently installed by Porter will be provided, along with metadata such as creation time, last action, last status, etc.
```

Comment:
Can you hard wrap the help text line here so that it fits into the text box on the doc page? You can see about where I was wrapping it for the other commands.

Otherwise people have to sidescroll to read the text which is awkward.

Here's the preview of your helptext on the website so you can see what I'm talking about: https://deploy-preview-350--porter.netlify.com/cli/#bundles-list, if you run `make docs-preview` you can try it out locally first.

Reply:
Updated w/ a hard wrap and also added an entry to the command in the `config.toml` file.